Fix example for "continue-at" (#62)
curl/wcurl: wcurl.md

**wcurl --curl-options="--progress-bar --http2" example.com/filename.txt**

> Resume from an interrupted download, if more options are used, these ones needs
> to be the last one in the list. Note that the only way to resume interrupted
> downloads is to allow wcurl to overwrite the destination file:

Reviewer: ```suggestion
* Resume from an interrupted download. The options necessary to resume the download (`--clobber --continue-at -`) must be the **last** options specified in `--curl-options`. Note that the only way to resume interrupted downloads is to allow wcurl to overwrite the destination file:
```
